.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: node_js
 node_js:
-  - "8"
```

Comment:
Please re-add this unless this doesn't run on `node@8`? `node@8` is Maintenance LTS until 12/2019 ([source](https://github.com/nodejs/Release/#release-schedule)).